client/src/test/java/com/alibaba/nacos/client/ConfigTest.java

```diff
@@ -29,52 +30,76 @@
 
 import java.util.Properties;
 import java.util.Scanner;
+import java.util.concurrent.atomic.AtomicBoolean;
 
 @Ignore
 public class ConfigTest {
-    
+
```

Comment:
Please do not change the indent.

Please use nacos CodeStyle to reformat code

Reply:
Already fixed it and reformatted code  using  nacos CodeStyle.